### PR TITLE
[SECURITY-1014] Match vulnerable Libvirt Agents

### PR DESCRIPTION
**Pull request by bgermann**
The SECURITY-1014 vulnerabilities were fixed in Libvirt Agents plugin
version 1.8.6.  Match only the vulnerable versions in order to warn.

### CODE REVIEW

File: resources/warnings.json [hunk 1]
```diff
@@ -6377,7 +6377,7 @@
     "versions": [
       {
         "lastVersion": "1.8.5",
-        "pattern": ".*"
+        "pattern": "1\\.[0-7].*|1\\.8.[0-5]"
```
Comment (daniel-beck) [inline resources/warnings.json:6380]: Better never release version 1.10.

I recommend you model your pattern after the existing ones. My recommendation:

    (1[.]7|1[.]8[.][1345])(|[.-].+)|(1[.]8)

Comment (bgermann) [reply resources/warnings.json]: Okay, I updated the pattern according to your suggestion.

**Comment by daniel-beck** [comment]
@bgermann The CSRF vulnerability that is part of SECURITY-1014 (2) has not been resolved.

Please see the documentation at https://www.jenkins.io/doc/developer/security/form-validation/

**Comment by bgermann** [comment]
Okay, let me fix that. I'll open a new PR once it is fixed.

**Pull request closed without merging**